### PR TITLE
[plugin/webMonitor] Show current player on sidebar's map

### PR DESCRIPTION
**Pull request by BubonicPestilence**
Preview: https://yadi.sk/i/WjiWckwF3KRqdf

### CODE REVIEW

File: plugins/needs-review/webMonitor/trunk/WWW/_sidebar.html.template [hunk 1]
```diff
@@ -2,18 +2,111 @@
...
+                var newWH = {
+                    width: "",
+                    height: "",
+                };
+                
+                if ($map.css("width") != "512px") {
+                    newWH = {
+                       width: "512px", 
+                       height: "512px", 
+                    };
```
Comment (BubonicPestilence) [inline plugins/needs-review/webMonitor/trunk/WWW/_sidebar.html.template]: Yeah, currently i stuck with conversion of original coorditaions to map files. Annyn suggestions?

P.S. My local version works with party as well :)

**Comment by kaliwanagan** [comment]
Merging this in anyway; we can just fix the width and height issue later.

File: plugins/needs-review/webMonitor/trunk/WWW/_sidebar.html.template (hunk 1 above)
Comment (alisonrag) [inline plugins/needs-review/webMonitor/trunk/WWW/_sidebar.html.template]: in interface wx we have the conversion of original to map, we need just search for the code

**Comment by BubonicPestilence** [comment]
I'll do it this week, if none else do it faster, just busy atm

**Pull request merged**